settings.gradle

```diff
@@ -28,6 +28,10 @@ include 'fabric-loot-tables-v1'
 include 'fabric-mining-levels-v0'
 include 'fabric-models-v0'
 include 'fabric-networking-v0'
+include 'fabric-networking-v1'
+include 'fabric-networking-handshake-v1'
+include 'fabric-networking-player-tracking-v1'
+include 'fabric-network-testmod'
```

Comment:
This is fine for now, but dont ship with this. I will try and find time to start reworking all the test mods.

Reply:
Yes this is just a dummy so that I can run the test mod for now. should remove it later.